package.json

```diff
@@ -60,7 +60,7 @@
     "moment-timezone": "0.5.17",
     "ng-tags-input": "3.2.0",
     "re-tree": "0.0.2",
-    "restangular": "1.3.1",
+    "restangular": "^1.6.1",
```

Comment:
We should use the fixed version for "production" dependency

Reply:
oh yes

Reply:
fixed